### PR TITLE
fix: [M3-7151] - Redis trademark changes

### PR DESCRIPTION
**Pull request by hmorris3293**
## Description 📝
Added Redis Legal verbiage to the app descriptions and any instance of Redis includes the Trademark logo

## Preview 📷

image

| Single node  | Cluster   |
| ------- | ------- |
| image | image |

## How to test 🧪
- Go to `/linodes/create?type=One-Click`
- Ensure mentions of redis include a trademark
- Ensure searching for redis still displays the two apps
- Ensure the drawer still opens and that there is a legal disclaimer in the drawer
- Redis logo should be replaced by 3 generic stacked slices

### CODE REVIEW

**Review by bnussman-akamai — APPROVED** [review]
Looks good! 

Does this have to been in quotes? Feels a bit weird to me

Screenshot 2023-09-18 at 3 05 36 PM

**Review by mjac0bs — COMMENTED** [review]
Are these fields also in need of the trademark in the Marketplace App for Redis® Sentinel Cluster Setup?
image


Confirmed: 
✅ Mentions of Redis® include a trademark
✅ Searching still displays the two apps (both by search bar and by category - `Databases`)
✅ Drawer still opens and there is a legal disclaimer in the drawer - had a minor question about the quotes around the legal disclaimer, but nothing to hold up the release
✅ Logo should be replaced by 3 generic stacked slices
✅ Guide link redirects as expected

File: packages/manager/src/features/OneClickApps/oneClickApps.ts [hunk 1]
```diff
@@ -1944,14 +1944,15 @@ export const oneClickApps: OCA[] = [
       end: '722b20',
       start: '222222',
     },
-    description: `Redis is an open-source, in-memory, data-structure store, with the optional ability to write and persist data to a disk, which can be used as a key-value database, cache, and message broker. Redis features built-in transactions, replication, and support for a variety of data structures such as strings, hashes, lists, sets, and others.`,
+    description: `Redis&reg; is an open-source, in-memory, data-structure store, with the optional ability to write and persist data to a disk, which can be used as a key-value database, cache, and message broker. Redis&reg; features built-in transactions, replication, and support for a variety of data structures such as strings, hashes, lists, sets, and others.<br/><br/>"*Redis is a registered trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by Akamai Technologies is for referential purposes only and does not indicate any sponsorship, endorsement or affiliation between Redis and Akamai Technologies."`,
```
[the remark this answers is not in the document]
Comment (hana-linode) [reply packages/manager/src/features/OneClickApps/oneClickApps.ts:1947]: Removed, I don't think it's needed. There will be a follow up PR in the future to refine everything once we get further feedback from Redis

**Comment by hana-linode** [comment]
@mjac0bs Holden will look into the Setup inputs asynchronously

**Review by hana-linode — APPROVED** [review]

**Pull request merged**